docker/.env

```
@@ -0,0 +1,3 @@
+PG_PASS=PDJ7RxQHJm5WYcDlnrWC740ygFDo5t2KrtUe0Y9l73VuUnNm
+AUTHENTIK_SECRET_KEY=dUAilaQwaKYTjXZPSkWaV2Wql5rFVOy+cVpfv5U77Ql3jKP0l49eQae2BmYxbbUH5VFwYcF2OHXbHtIv
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTHENTIK_SECRET_KEY key should go before the PG_PASS key

(UnorderedKey)